### ISSUE TITLE
Fonts partly working in MS Office

### ISSUE THREAD

**Issue by rkayzer** [graphicore/librebarcode#24]
I downloaded the code 39 fonts from fonts.google.com, and installed them in Windows 10.
So far so good. Using them in Libreoffice work fine, in MS Office 2007 also.
However, using MS office 2013 and 2019 I can only get the fonts to work in Excel, not in Word.
In Word I can select the font, but *12345* is still displayed as human readable *12345*, not as a barcode.
Any suggestion on how to fix this is greatly appreciated. 
BTW where it is working it seems to be working very good, no scanning problems :-).

**Comment by graphicore** [comment]
@rkayzer thanks for reporting! Is this maybe a duplicate of #18?

I can't currently fix it, but I have it on my list. Should happen before #23 I believe. By the way even though code 39 works nice for you, there's a bug in the version you are using, see #21. The fixed versions have not been released yet, but code 39 seems to be robust enough anyway. :grin:

**Comment by graphicore** [comment]
Could be Unicode range bits setting.

**Comment by rkayzer** [comment]
Thank you for fast response.
As mentioned above this is indeed a duplicate of bug ? I have tried the updated font mentioned in the base bug, and it installs properly and seems to be working correct (tested using *123456* size 36 and then scanning it from my screen using Android phone app).
